addons/event_sale/views/event_registration_views.xml

```diff
@@ -22,15 +22,9 @@
         <field name="model">event.registration</field>
         <field name="inherit_id" ref="event.event_registration_view_kanban"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='event_attendees_kanban_icons_desktop']" position="inside">
-                <div class="mt-auto pt-2">
-                    <field name="sale_status"/>
-                </div>
-            </xpath>
-            <xpath expr="//div[@id='event_ticket_id']" position="before">
-                <div class="d-md-none" >
-                    <field name="sale_status"/>
-                </div>
+        <field name="active"/>
```

Comment:
miss an indentation and also why do we need this one exactly and don't we need a position set ? (lot of question for an active field 😆 )

Reply:
probably a copy paste done too quick :upside_down_face: